examples/tensorflow/object_detection/main.py

```diff
@@ -384,6 +380,8 @@ def run(config):
     test_step = create_test_step_fn(strategy, compress_model, predict_post_process_fn)
 
     if "train" in config.mode:
+        if config.weights is None:
+            logger.warning("Pretrained checkpoint is not provided. This may lead to poor training results!")
```

Comment:
What is about resume mode? Will the user see this warning?

Reply:
The user will see this warning in resume mode

Reply:
It seems like this behavior is wrong, doesn't it?

Reply:
Seems like it, because in resume mode the model and compression state are loaded from `ckpt_path` argument and not `weights` argument. Added a check for resume mode to the if.